addons/account/models/account_account.py

```diff
@@ -1011,16 +1002,8 @@ def write(self, vals):
                 if self.env['account.move.line'].search_count([('account_id', '=', account.id), ('currency_id', 'not in', (False, vals['currency_id']))]):
                     raise UserError(_('You cannot set a currency on this account as it already has some journal entries having a different foreign currency.'))
 
-        code_mapping_ids = vals.pop('code_mapping_ids', None) or []
```

Comment:
We still need to handle the situation where the user imports a CoA and doesn't fill the `code_mapping_ids/company_id` and `code_mapping_ids/code` columns for some rows. At the moment, this triggers a create/write with `{'code_mapping_ids': False}` in the vals.

It was to prevent that `False` value from triggering a `Command.clear()` that I did `vals.pop('code_mapping_ids', None) or []` in both the create and the write. 

Perhaps this isn't the right place to do it, but we definitely need to ensure somewhere that writing/creating with {'code_mapping_ids': False} doesn't do anything, as that would be a non-sensical operation.

Reply:
Didn't try it but there is no unlink defined on mappings, and the access rights don't allow unlinking either

Reply:
Note that writing `{'code_mapping_ids': False}` won't try to `unlink` the existing `account.code.mapping`, it will just try to set their `account_id` to `False`. 
But still, we should neutralize it before it happens - I think that's the reason the runbot is red at the moment :smile: